android/core/src/main/java/com/mongodb/stitch/android/core/auth/StitchAuth.java

```diff
@@ -98,4 +99,20 @@
    * @param listener the listener to remove.
    */
   void removeAuthListener(final StitchAuthListener listener);
+
+  /**
+   * Returns a list of all logged in users.
```

Comment:
[task] add comments to the other methods in this file like `getUser` to clarify that those are operating on the "active" user